main_test.go

```
@@ -0,0 +1,30 @@
+
+func TestMain(t *testing.T) {
+	g := Goblin(t)
+
+	postgresC, err := tests.Setup(t)
+	if err != nil {
+		t.Fatalf("failed to setup postgres container: %s", err)
+		return
+	}
+
+	dbConnString, err := tests.GetConnString(t, postgresC)
+	db.ConnectDB(dbConnString)
```

golangci-lint

Error return value of `db.ConnectDB` is not checked

(errcheck)